### PR TITLE
Create a RC package

### PR DESCRIPTION
**Pull request by melund**

### CODE REVIEW

**Comment by bollwyvl** [comment]
> The top level meta key channel_targets is unexpected

Goes in `conda_build_config.yaml` a la https://github.com/conda-forge/jupyterlab-feedstock/blob/rc/recipe/conda_build_config.yaml

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,13 +27,15 @@ requirements:
...
     - python >=3.7
+    - sphinx-theme-builder >=0.2.0a7
   run:
     - beautifulsoup4
     - docutils !=0.17.0
     - packaging
     - pygments >=2.7
     - python >=3.7
     - sphinx >={{ min_sphinx }}
+    - accessible-pygments
```
Comment (bollwyvl) [inline recipe/meta.yaml]: please sort

**Comment by melund** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% set version = "0.12.0" %}
+{% set version = "0.13.0rc4" %}
 {% set cov_fail_under = "82" %}
```
Comment (bollwyvl) [inline recipe/meta.yaml]:
```
Required test coverage of 82% reached. Total coverage: 85.28%
```

Let's crank the coverage baseline to 85

Comment (melund) [reply recipe/meta.yaml]: No problem. I have increased it. But I don't think the coverage testing belongs here in this repo :)

**Comment by bollwyvl** [comment]
Thanks. It _won't_ be automatic to get new releases out: I watch the upstream releases, but feel free to ping if we're slacking.

File: recipe/meta.yaml (hunk 2 above)
Comment (bollwyvl) [reply recipe/meta.yaml]: A bit of context: as a maintainer of several hundred feedstocks: running the test suite, under coverage, provides a first line of defense against optional (but important for whatever reason) dependency drift.

A package known to be under full test is much safer to have `automerge` enabled (as this one does, though I take issue with that #29). Fixing a few deps, and assessing changes when things fail is much less work, at scale, and crucially requires less total intervention than waiting for an end user to have a bad day, maybe complaining to the upstream, finally landing here, making a patch release, maybe making a repodata patch, etc.

Comment (melund) [reply recipe/meta.yaml]: Ahh Thanks. I hadn't thought about that, but I totally see your point.  That is also why I wanted the RC packages for this :)

**Pull request merged**